tests/libtest/codechecker.py

```diff
@@ -421,7 +425,7 @@ def start_or_get_server():
 def wait_for_server_start(stdoutfile):
     print("Waiting for server start reading file " + stdoutfile)
     n = 0
```

Comment:
There is nothing in the loop below that would wait for **n** seconds... this `n` here is a try counter with a linear 1 sec re-try loop.